week-5-practise/js/main.js

```diff
@@ -0,0 +1,66 @@
+//// PART-1 ////
+
+var placeToJumbotron = document.querySelector(".jumbotron");
+var placeToDonateABike = document.querySelector(".btn-primary"+".btn-lrg");
```

Comment:
Just a minor detail: What's the `"+"` for? I think it would be easier to read without it.

Reply:
My code doesn't work without "+".

Reply:
If you take off all three characters (`"+"`) it will work.